jdbc-adapter/virtualschema-jdbc-adapter/src/test/resources/integration/vschema_complex_inner_join.json

```diff
@@ -0,0 +1,240 @@
+{
+  "testSchema" :
+  [
+    "create table t1(id int, val int)",
```

Comment:
Use upper case for consistency, please.

Reply:
We have no convention to write SQL keywords in uppercase.